package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tiny-frontend/tiny-client-react",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
As we haven't released that package yet I think we can also not upgrade the version and keep 0.0.1 😊

Reply:
Agreed, done ✅